services/skipchain/skipchain.go

```diff
@@ -443,30 +443,36 @@ func (s *Service) bftVerify(msg []byte, data []byte) bool {
 				return true
 			}
 		}
+	default:
+		f, ok := verifiers[sb.VerifierID]
```

Comment:
I left it here because it uses fields of the structure, but it makes sense what you say. It will be a bit clumsy, but probably nicer...